contact_info_service.py

```
@@ -0,0 +1,141 @@
+            Exception: If openai_client is None.
+        """
+        if openai_client is None:
+            raise Exception("openai_client must be provided")
+
+        self.open_ai_client = openai_client
+
+    def __get_user_prompt(self, business_name: str) -> str:
+        return f"""
+        Find the official contact information for the company named "{business_name}".
+    
+        You must determine the company’s:
```

Ruff (0.14.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)